src/compiler/emitter.ts

```diff
@@ -19,6 +19,14 @@ module ts {
         return isExternalModule(sourceFile) || isDeclarationFile(sourceFile);
     }
 
+    // flag enum used to request and track usages of few dedicated temp variables
+    // enum values are used to set/check bit values and thus should not have bit collisions.
+    const enum TempVariableKind {

```

Comment:
/* @flags */ :)   At least to indicate that next value should be '4' and not '3'.